README.md

```diff
@@ -1,2 +1,105 @@
 # Bhagavad-Gita-App
 Flutter app for Gita
+<h1 align="center">Getting Started with Bhagavad Gita App </h1> 
+<br>
+
+<div align="center">
+
+[![Welcome to my profile](https://img.shields.io/badge/Hello,Programmer!-Welcome-blue.svg?style=flat&logo=github)](https://github.com/Ggita/Bhagavad-Gita-App)
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/gita/Bhagavad-Gita-App)
+![License](https://img.shields.io/badge/License-MIT-red)
+![Stars](https://img.shields.io/github/stars/gita/Bhagavad-Gita-App?style=flat&logo=github)
+![Forks](https://img.shields.io/github/forks/gita/Bhagavad-Gita-App?style=flat&logo=github)
+
+</div>
+
+<div align="center">
+  <img height=240 src="../Bhagavad-Gita-App-1/bhagavad_gita/assets/images/gita.png" alt="Bhagavad-Gita-Image">
+</div>
+
+<br>
+
+<div align="center">
+<img src ="https://forthebadge.com/images/badges/winter-is-coming.svg"/>
+  <img src="https://forthebadge.com/images/badges/for-you.svg" />
+
+  <img src="https://forthebadge.com/images/badges/check-it-out.svg" alt="check">
+  <img src="https://forthebadge.com/images/badges/built-by-developers.svg" />
+</div>
```

Comment:
This is not required I guess, also makes it weird for readme according to me, please remove this and we should be good to merge the PR,
Thanks for your efforts

Reply:
Ok